SPECS/kernel-mshv/config

```diff
@@ -6735,8 +6735,8 @@ CONFIG_LSM="lockdown,yama,loadpin,safesetid,integrity,apparmor,selinux,smack,tom
 # Memory initialization
 #
 CONFIG_INIT_STACK_NONE=y
-CONFIG_INIT_ON_ALLOC_DEFAULT_ON=y
-CONFIG_INIT_ON_FREE_DEFAULT_ON=y
+#CONFIG_INIT_ON_ALLOC_DEFAULT_ON=y is not set
+#CONFIG_INIT_ON_FREE_DEFAULT_ON=y is not set
```

Comment:
```suggestion
# CONFIG_INIT_ON_ALLOC_DEFAULT_ON is not set
# CONFIG_INIT_ON_FREE_DEFAULT_ON is not set
```

Reply:
I didn't know you can suggest changes like that. I just had to press commit button for your suggested changes. Really cool!